worlds/hk/docs/setup_en.md

```diff
@@ -4,14 +4,11 @@
 * Download and unzip the Scarab+ Mod Manager from the [Scarab+ website](https://themulhima.github.io/Scarab/).
 * A legal copy of Hollow Knight.
 
-## Optional Software
-* Archipelago Map Mod from Scarab+
-  * Ensure that both RandoMapMod and MapChanger are uninstalled or disabled as they are incompatible with Archipelago Map Mod.
-
-## Installing the Archipelago Mod using Scarab
+## Installing the Archipelago Mod using Scarab+
 1. Launch Scarab+ and ensure it locates your Hollow Knight installation directory.
 2. Click the "Install" button near the "Archipelago" mod entry.
-3. Launch the game, you're all set!
+   * If desired, also install "Archipelago Map Mod" to use as an in-game tracker.
+4. Launch the game, you're all set!
```

Comment:
there is no 3.? Underwear gnomes not ok with this change.

Reply:
Hm, idk what happened there
However it is markdown so the number in front of the . actually doesn't matter at all when rendering. I'd personally be more inclined to renumber everything to 1. than to correct the numbering, if that's alright

Reply:
Because lazy numbering is awful when you read something like a longer list of instructions in source form. Being readable in source form is one of the features of markdown.

Reply:
I will make the change but I don't think that rationale holds up

1. Who do you expect to be actually reading the instructions in source form
2. When does the actual number of numbered steps ever matter, unless you say "refer back to step X?" The only purpose of a numbered list is to indicate to do the steps in that order, even if you are reading in source the fact that the list is numbered at all, regardless of the actual number, still indicates that

Reply:
The non-lazy numbering existed before I joined and was documented to be our standard in style.md because it was already actively being used that way (and consistency is a good thing). I'll still try to defend my personal opinion below:

1. whenever you use a project-wide search function (e.g. on github, in pycharm, ack or grep); also while modifying the code; there might also be a report that "step 4 is wrong"
2. when referencing the step - while helping, when reporting an issue or as back ref in the same document

Either way, 1., 2., 4. is definitely neither fine in lazy nor non-lazy numbering.